projects/demo/src/modules/charts/axes/axes.component.ts

```diff
@@ -47,11 +47,11 @@ export class ExampleTuiAxesComponent {
         'hidden',
     ];
 
-    readonly labelsVariants = [
+    readonly labelsVariants: readonly (readonly string[])[] = [
         [],
         ['', '25%', '50%', '100%'],
         ['One', 'Two', 'Three'],
-        ['One', null, '', 'Two and a half', 'Three', null],
```

Comment:
Are you sure `null` and `''` are the same? There specifically were both cases in the array here.

Reply:
fixed